components/Pages/LandingPage/LandingPage.tsx

```diff
@@ -50,30 +55,105 @@ const LandingPage = () => {
+            <li>
+              Copyright Registration: Although you automatically own the
+              copyright to your music upon creation, registering it with the
+              U.S. Copyright Office (or the equivalent in your country) provides
+              additional legal protection.
+            </li>
+          </ul>
+          Summary
+          <br />
+          By self-publishing and self-distributing, you keep all the creative
+          and financial control over your music but take on the added
+          responsibility of managing rights, royalties, promotion, distribution
+          logistics and financial distribution. You need to be well-organized
+          and either learn how to handle these aspects, hire professionals or
+          use online tools to assist you.
+        </div>
+      </InfoDialog>
 
-      <div className="mt-10 w-full flex md:justify-start justify-center">
+      <InfoDialog
+        isOpen={isDisclaimerOpen}
+        close={() => setIsDisclaimerOpen(false)}
+      >
+        <p className="mb-4">
+          <b>DISCLAIMER:</b> Our intention is to provide a platform for
+          self-help. We provide a draft of a contract that should be
+          meticulously reviewed by each of the parties willing to make an
+          agreement. The information given in this service is provided for your
+          private use and does not constitute legal advice. We do not review any
+          information you provide us for legal accuracy or sufficiency, draw
+          legal conclusions, provide opinions about your usage, or apply the law
+          to the facts of your situation.
+        </p>
+        <p className="mb-4">
+          If you don’t understand the terms or consequences of the draft we
+          provide, or need legal advice for a specific problem, we encourage you
+          to consult with a licensed attorney. The draft provided by this
+          service is not a substitute for legal advice from a qualified attorney
+          licensed to practice in an appropriate jurisdiction.
+        </p>
+        <p>
+          This draft is limited to the general principles of copyright law. If
+          you are interested in learning more about how the MESA system can help
+          you, please get in touch at <b>contact@mesawallet.io</b>.
+        </p>
         <button
           type="button"
-          className="bg-grey-dark border-2 border-danger font-rubik text-lg rounded-lg py-2 px-6 md:w-auto h-24 md:h-14 w-72"
+          className="flex ml-auto items-center bg-grey-dark border-2 border-white rounded-lg py-2 px-6 mt-10"
           onClick={() => push('/contract-builder')}
         >
-          GET STARTED
+          <span>Proceed</span> <ArrowRightIcon className="size-4 ml-2" />
         </button>
-      </div>
-
-      <div className="md:hidden mt-6">
-        <ReadHereLink className="text-[15px] text-center" link="/" />
-      </div>
+      </InfoDialog>
     </div>
-
   )
 }
```

Comment:
component is too large.
- please break this monolithic component into smaller, more reusable, components.
- actual: file size is `more than 100 lines of code.`
- expected: file size is less than or equal to `99 lines` of code.